### PR TITLE
[fix] disable publish button until image upload were completed

### PR DESCRIPTION
**Pull request by rado196**
Publish buttons must be disabled until image uploads were completed.

### CODE REVIEW

File: clients/app/src/views/screens/CreateFanClub/screens/CreateFanClub/containers/SwiperContainer.tsx [hunk 1]
```diff
@@ -8,13 +8,19 @@ import { useTheme } from 'themes';
 import Icon from 'views/components/base/Icon';
 import { Icons } from 'assets/icons';
 import { fanClubsStore } from '../../../store';
```
Comment (hay92k) [inline clients/app/src/views/screens/CreateFanClub/screens/CreateFanClub/containers/SwiperContainer.tsx:10]: please replace '../../../

File: clients/app/src/views/screens/CreateFanClub/screens/CreateFanClub/containers/VisualContainer.tsx [hunk 2]
```diff
@@ -1,10 +1,32 @@
+/* eslint-disable react-hooks/exhaustive-deps */
+
 import { AwsS3FolderEnum } from '@ultras/utils';
 import React from 'react';
 import { Text, VStack } from 'native-base';
 import I18n from 'i18n/i18n';
 import ChoosePhotoComponent from '../components/ChoosePhotoComponent';
+import { useFileUploadStore } from '../../../fileUploadStore';
```
Comment (hay92k) [inline clients/app/src/views/screens/CreateFanClub/screens/CreateFanClub/containers/VisualContainer.tsx:8]: same here

**Review by hay92k — APPROVED** [review]

File: clients/app/src/views/screens/CreateFanClub/screens/CreateFanClub/containers/SwiperContainer.tsx (hunk 1 above)
Comment (rado196) [reply clients/app/src/views/screens/CreateFanClub/screens/CreateFanClub/containers/SwiperContainer.tsx]: imports updated!

File: clients/app/src/views/screens/CreateFanClub/screens/CreateFanClub/containers/VisualContainer.tsx (hunk 2 above)
Comment (rado196) [reply clients/app/src/views/screens/CreateFanClub/screens/CreateFanClub/containers/VisualContainer.tsx]: imports updated!

**Review by hay92k — APPROVED** [review]

**Pull request merged**